### PR TITLE
GDPR compliant privacy  policy

### PR DESCRIPTION
**Pull request by MHerbst**
The new GDPR compliant privacy policy

### CODE REVIEW

File: privacy.md [hunk 1]
```diff
@@ -3,20 +3,286 @@ layout: AboutPage
...
+However, we wish to point out that doing so may mean you will not be able to enjoy the full
+functionality of this website. You can also prevent the data generated by cookies about your use
+of the website (incl. your IP address) from being passed to Google, and the processing of these
+data by Google, by downloading and installing the browser plugin available at the following link: https://tools.google.com/dlpage/gaoptout?hl=en.
+
+####  Objecting to the collection of data
+
+You can also prevent the collection of your data by Google Analytics by clicking on the following
+link. An opt-out cookie will be set to prevent your data from being collected on future visits to
+this site: <a href="javascript:gaOptout();">Disable Google Analytics</a>.
```
Comment (ghys) [inline privacy.md]: This will probably not work, I'll come up with something to address it.

File: privacy.md [hunk 2]
```diff
@@ -3,20 +3,286 @@ layout: AboutPage
...
+request is sufficient. The data processed before we receive your request may still be
+legally processed.
+
+### Right to file complaints with regulatory authorities
+
+If there has been a breach of data protection legislation, the person affected may file
+a complaint with the competent regulatory authorities. The competent regulatory authority
+for matters related to data protection legislation is the data protection officer of the
+German state in which our company is headquartered. A list of data protection officers
+and their contact details can be found at the following link: https://www.bfdi.bund.de/DE/Infothek/Anschriften_Links/anschriften_links-node.html.
```
Comment (ghys) [inline privacy.md]: These URLs are not transformed into hyperlinks, I'll fix them later.

Comment (MHerbst) [reply privacy.md]: @ghys Sorry about the URLs. The problem with the URLs were caused by my Markdown previewer. It automatically converted them into links, so I forgot to format them correctly :-(.

- Google Analytics: I think you can use the same code. Please make sure that the anonymize_ip option is set to true.
- This is a good question. If we use this page also for the community you can simply add the text from section "Registration on this Website" from this document: https://docs.google.com/document/d/1No9_GC6urV7pkKR5KlSsovJI4bX6TS9XG6OoOKUOjLU/edit?usp=sharing

**Comment by kaikreuzer** [comment]
> I suppose it's ok to re-enable it with the same ID 

Yes!

**Comment by ghys** [comment]
Looks good, I also have enabled GA with anonymizeIp and the gaOptout() function in the previous commit. Can't test properly now since my Pi-Hole is blocking it ;)

**Pull request merged**